net.efaxgtk.efaxgtk.yaml

```diff
@@ -0,0 +1,70 @@
+app-id: net.efaxgtk.efaxgtk
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: efax-gtk
+rename-desktop-file: efax-gtk.desktop
+rename-icon: efax-gtk
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=session-bus
+  - --share=network
+  - --filesystem=host
+  # For file browse dialogs
+  - --filesystem=xdg-run/gvfsd
+  # For modems
+  - --device=all
+cleanup:
+  - "/include"
+  - "*.la"
+  - "*.a"
+  - "/lib/debug"
```

Comment:
lib debug shouldn't be removed. it will be automatically put into the debug package.